README.md

```diff
@@ -327,7 +327,7 @@ We'll use `zq` to take an input, do no processing, and display it
 in pretty-printed Zed.
 
 ```
-echo "..." | zq -Z -
+echo '{"hello": "world"}' | zq -Z -
```

Comment:
In the unlikely event someone cut & pasted the `"..."` example as is, they'd get an error. I figure it doesn't hurt things to have it "real" but super simple.